distage/distage-framework-docker/src/main/scala/izumi/distage/docker/bundled/PostgresFlyWayDocker.scala

```diff
@@ -0,0 +1,80 @@
+package izumi.distage.docker.bundled
+
+import distage.Id
+import izumi.distage.docker.Docker.{DockerPort, DockerReusePolicy, Mount}
+import izumi.distage.docker.healthcheck.ContainerHealthCheck
+import izumi.distage.docker.{ContainerDef, ContainerNetworkDef}
+import izumi.distage.model.definition.ModuleDef
+import izumi.reflect.TagK
+
+object PostgresFlyWayDocker extends ContainerDef {
+  val primaryPort: DockerPort = DockerPort.TCP(5432)
+
+  override def config: Config = {
+    Config(
+      image = "library/postgres:12.3",
+      ports = Seq(primaryPort),
+      env = Map("POSTGRES_PASSWORD" -> "postgres"),
+      healthCheck = ContainerHealthCheck.postgreSqlProtocolCheck(primaryPort, "postgres", "postgres"),
+    )
+  }
+
+  object FlyWay extends ContainerDef {
+    override def config: Config = Config(
+      image = "flyway/flyway:6.0-alpine",
+      ports = Seq.empty,
+      cmd = Seq(
+        "-url=jdbc:postgresql://localhost/postgres",
+        "-schemas=public",
+        "-user=postgres",
+        "-password=postgres",
+        "-connectRetries=60",
+        "baseline",
+        "migrate",
+      ),
+      mounts = Seq(Mount(this.getClass.getClassLoader.getResource("sql").getPath, "/flyway/sql")),
```

Comment:
This will not work when resources are in jars (e.g. outside of dev env)
I think we need to allow parameterization to allow user to choose a suiteable location to bind...

Reply:
Yeah, I was thinking about the creation of a container with the configuration for this purpose. But decided to implement a hardcoded version to unify with the other containers style (which are objects) and wait till review. Because current independent Postgres container implementation also contains hardcoded values and hardly can be used in real-world tests.

Reply:
I'll update this container then.

Reply:
@neko-kai Done.